Engine/Component/ComponentTransform2D.h

```diff
@@ -0,0 +1,42 @@
+#ifndef _COMPONENTTRANSFORM2D_H_
+#define _COMPONENTTRANSFORM2D_H_
+#include "Component.h"
+class ComponentTransform2D : public Component
+{
+public:
```

Comment:
Because this class is similar to `ComponentTransform`, we possibly can implement it in a cleaner way making it child of `ComponentTransform`.